Users/coafile.rst

```diff
@@ -39,6 +39,15 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually
+don't want to lint like minified files
+and backup files (e.g. ``*.min.js``, ``*.orig``)
+such as the file present in ``.coafile`` in the main
```

Comment:
files?

Reply:
which line?

Reply:
whats the exact change u are asking to make?

Reply:
such as the files..

Reply:
`file` => `files` right?

Reply:
precisely